src/services/job.service.js

```diff
@@ -11,23 +12,91 @@ const createJob= async (jobBody,userId)=>{
   return {
     "status":true,
     "message": "Successfully posted job",
+    "jobId": job._id,
   };
 }
 
 const getJobs=async (page)=>{
-  const jobs= await Job.find().skip(page).limit(parseInt(PageDefaultLimit));
+  const jobs= await Job.find({'status':'Approved',paymentStatus:true}).skip(page).limit(parseInt(PageDefaultLimit));
   return jobs;
 }
 const getJobsById=async(user,page) =>{
   return Job.find({createdBy:user}).limit(parseInt(PageDefaultLimit)).skip(page);
 }
 const getRecomendedJobs=async() =>{
-  return Job.find({ $or :[{productType: 'Diamond'},{productType: 'Platinum'}]});
+  return Job.find({
+    $and:[
+    { status:'Approved'},
+    {paymentStatus:true},
+    { $or :[
```

Comment:
this should be and right?

Reply:
Yes annaya it is 'and' condition i mentioned the same  condition: approved & payment true & ( platinum or diamond )